### ISSUE TITLE
The code does not match the implementation details in the paper

### ISSUE THREAD

**Issue by wwh13843218306** [med-air/DLTTA#8]
The paper mentions the feature channel dimensions are [16,32,64,128], in reality on the OCT dataset your code is [64, 64, 64, 64]。And you didn't mention the part about ANET and AENET in the paper.

**Comment by hz0567** [comment]
AENET is for ATTA method. For the feature channels, do you mean the feature dimension for the UNET block ?

**Comment by wwh13843218306** [comment]
> AENET用于ATTA方法。对于特征通道，您的意思是 UNET 块的特征维度吗？

Yes, I found that unet's feature dimensions don't match the paper。
Did I misunderstand something?

**Comment by hz0567** [comment]
Hi, I think you can modify the unet_dims argument to match the feature channels dimension. Or you can directly use the default configuration.
